docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md

```diff
@@ -75,49 +70,65 @@ Make sure that the following changes have been applied by checking your database
+| Discount.storeRelation       | property | created | src/Generated/Shared/Transfer/DiscountTransfer                     |
+| CalculatedDiscount.priority  | property | created | src/Generated/Shared/Transfer/CalculatedDiscountTransfer           |
+| DiscountGeneral.priority     | property | created | src/Generated/Shared/Transfer/DiscountGeneral                      |
 
 {% endinfo_block %}
 
-{% info_block warningBox "Verification" %}
-
-Make sure that the discount form has a priority field and the discounts table has a priority column.
+### 3) Add translations
 
-Make sure, that the existing discounts in the spy_discount DB table have priority set to 9999.
+Generate a new translation cache for Zed:
 
-Make sure that discounts are calculated according to their priorities:
-
-    1. Create a couple of percentage discounts with different priorities.
+```bash
+console translator:generate-cache
+```
 
-    2. Add items to the cart to fulfill discounts' requirements.
+{% info_block warningBox "Verification" %}
 
-    3. Check that discounts are applied in the correct order and the calculated discount total is correct.
+Make sure that all labels and help tooltips in the discount create and edit forms have English and German translations.
 
 {% endinfo_block %}
 
----
-
-
-### Add translations
 
-Run the following command to generate a new translation cache for Zed:
+### 4) Build Zed UI frontend
 
-1. Append glossary for the feature:
+Enable Javascript and CSS changes:
 
 ```bash
-console translator:generate-cache
+console frontend:zed:install-dependencies
+console frontend:zed:build
 ```
 
 {% info_block warningBox "Verification" %}
 
-Make sure that all labels and help tooltips in discount form has English and German translation.
+Make sure that the discount form has a priority field and discounts table has a priority column.
+
+Make sure that the existing discounts in the `spy_discount` DB table have priority set to `9999`.
+
+Make sure that discounts are calculated according to their priorities:
+1. Create a couple of percentage discounts with different priorities.
+2. Add items to the cart to fulfill discounts' requirements.
+3. Check that discounts are applied in the correct order and the calculated discount total is correct.
 
 {% endinfo_block %}
 
+{% info_block warningBox "Verification" %}
+
+Make sure that you can submit the *Discount Create* and *Discount Update* forms with specified date and time for **Valid From** and **Valid To** discount form fields:
+1. In the Back Office, go to **Merchandising** > **Discount**.
+2. Create a new discount or update an existing one, check that you can see the *Discount* form.
+3. Click on **Valid From** and **Valid To** fields to ensure you can see the calendar, where you can select specific date and time.
+4. Make sure **Valid From** and **Valid To** fields can accept the selected date and time, using keystroke entry for times not falling on the hour.
```

Comment:
With the date/time picker, you can only click on a time that is on the hour, for example 13:00. If you want to specify a time like 13:45, you would need to type it in. I wanted to make it clear that it was possible to select a time that wasn't exactly on the hour, hence this phrase. Feel free to adjust for clarity.